FiniteCocompletions/gap/FiniteStrictCoproductCocompletion.gd

```diff
@@ -47,11 +47,7 @@ DeclareAttribute( "FiniteStrictCoproductCocompletion",
 #! @InsertChunk LeftCartesianDistributivityExpanding
 #! @InsertChunk TerminalCategory_as_FiniteStrictCoproductCocompletion
 
-CapJitAddTypeSignature( "FiniteStrictCoproductCocompletion", [ IsCapCategory ], function ( input_types )
-    
-    return CapJitDataTypeOfCategory( FiniteStrictCoproductCocompletion( input_types[1].category ) );
-    
-end );
+CapJitAddTypeSignature( "FiniteStrictCoproductCocompletion", [ IsCapCategory ], IsFiniteStrictCoproductCocompletion );
```

Comment:
Does `IsFiniteStrictCoproductCocompletion` imply `IsCapCategory`? If yes, this is not a valid type signature because a concrete category would have to be given.

Reply:
Yes it does. I see. I probably made this mistake somewhere else. Thank you for catching it.